examples/seismic/elastic/operators.py

```diff
@@ -0,0 +1,80 @@
+
+from devito import Eq, Operator, TimeFunction, left, right, staggered_diff
+from examples.seismic import PointSource, Receiver
+
+
+def ForwardOperator(model, source, receiver, space_order=4,
+                    save=False, kernel='OT2', **kwargs):
+    """
+    Constructor method for the forward modelling operator in an acoustic media
```

Comment:
docstring to be refreshed?

Reply:
`kernel` keyword argument is unused, it should be dropped

Reply:
I also think we should put a reference to the equations that are being implemented here